src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -45,13 +46,34 @@
     <% } %>
 
     <% if(request.getSession().getAttribute("user") != null){ %>
-      <h1>New Conversation</h1>
+      <h1>Create Group Conversation</h1>
       <form action="/conversations" method="POST">
           <div class="form-group">
-            <label class="form-control-label">Title:</label>
+            <label class="form-control-label">Group Name:</label>
           <input type="text" name="conversationTitle">
-        </div>
+          </div>
+          <div class="form-group">
+            <label for="AccessControl">Set Access</label>
+            <select name="AccessControl" style="width:300px;">
+              <option value="Public">Public</option>
+              <option value="Private">Private</option>
+            </select>
+          </div>
+          <h4>(Only if Conversation is private)</h4>
```

Comment:
I believe we can do a real if statement here rather than just have the heading :)